### PR TITLE
feat(nav): adding services, prep work 

### PR DESCRIPTION
**Pull request by davidvader**
prep work for go-vela/community#166

this introduces services as a new page at `org/repo/14/services`

entrypoint to this page will be implemented with go-vela/community#166 with the introduction of navigation tabs

### CODE REVIEW

**Comment by davidvader** [comment]
creating as Draft, most of this PR is present in #252 and #253, will open for review when those have been merged.

File: src/elm/Pages/Build/Model.elm [hunk 1]
```diff
@@ -32,34 +37,72 @@ import Vela
...
-    | FocusOn String
+type alias Msgs msg =
+    { collapseAllSteps : msg
+    , expandAllSteps : ExpandAll msg
+    , expandStep : Expand msg
+    , collapseAllServices : msg
+    , expandAllServices : ExpandAll msg
+    , expandService : Expand msg
+    , logsMsgs : LogsMsgs msg
+    }
```
Comment (davidvader) [inline src/elm/Pages/Build/Model.elm:63]: now we can add services to our component msg bus
this is where moving Msg back to Main.elm is becoming extremely useful..... trial by fire 🔥

File: src/elm/Vela.elm [hunk 2]
```diff
@@ -395,6 +399,7 @@ type alias BuildModel =
     { buildNumber : BuildNumber
     , build : WebData Build
     , steps : StepsModel
+    , services : ServicesModel
     }
```
Comment (davidvader) [inline src/elm/Vela.elm:404]: this is why we moved Steps into the StepsModel

File: src/scss/_main.scss [hunk 3]
```diff
@@ -794,20 +798,24 @@ nav {
...
+.service .-info .-duration {
   font-family: var(--font-code);
 }
 
+.no-services {
+  padding: 1rem 1rem;
+
+  background-color: var(--color-bg-dark);
+}
+
```
Comment (davidvader) [inline src/scss/_main.scss:818]: more services-specific style will be coming with the navigation tabs

**Review by wass3r — APPROVED** [review]

**Review by jbrockopp — APPROVED** [review]
LGTM

**Pull request merged**